README.md

```diff
@@ -1,3 +1,112 @@
-# tdr-terraform-environments
+# TDR Terraform Environments
 
-Readme for tdr-terraform-environments repository
+This respository contains the Terraform code to create the AWS resources needed to support the TDR application.
+
+## Terraform Structure
+
+The prototype is divided into separate Terraform modules that represent the different AWS resources that are needed to for the TDR project.
+
+The different modules are used by Terraform workspaces which represent three AWS environments:
+
+* intg
+* staging
+* prod
+
+## Getting Started
+
+### Terraform Backend
+
+Ensure that the Terraform backend has been created.
+
+See here: https://github.com/nationalarchives/tdr-dev-documentation/blob/tkAWSInfrastructureSetup/tdr-create-aws-instructure-setup.md
```

Comment:
The docs in that link should probably go in the manual folder, so we can keep all the general "how to" info in one place: https://github.com/nationalarchives/tdr-dev-documentation/tree/master/manual

It would also be good to describe the difference between the two Terraform projects. I know I keep getting confused by them. :)

Reply:
resolved